provider/cmd/pulumi-resource-hcloud/schema.json

```diff
@@ -3920,7 +3920,7 @@
-            "description": "Provides a Hetzner Cloud SSH key resource to manage SSH keys for server access.\n\n## Example Usage\n\n\u003c!--Start PulumiCodeChooser --\u003e\n```typescript\nimport * as pulumi from \"@pulumi/pulumi\";\nimport * as fs from \"fs\";\nimport * as hcloud from \"@pulumi/hcloud\";\n\n// Create a new SSH key\nconst _default = new hcloud.SshKey(\"default\", {publicKey: fs.readFileSync(\"~/.ssh/id_rsa.pub\", \"utf8\")});\n```\n```python\nimport pulumi\nimport pulumi_hcloud as hcloud\n\n# Create a new SSH key\ndefault = hcloud.SshKey(\"default\", public_key=(lambda path: open(path).read())(\"~/.ssh/id_rsa.pub\"))\n```\n```csharp\nusing System.Collections.Generic;\nusing System.IO;\nusing System.Linq;\nusing Pulumi;\nusing HCloud = Pulumi.HCloud;\n\nreturn await Deployment.RunAsync(() =\u003e \n{\n    // Create a new SSH key\n    var @default = new HCloud.SshKey(\"default\", new()\n    {\n        PublicKey = File.ReadAllText(\"~/.ssh/id_rsa.pub\"),\n    });\n\n});\n```\n```go\npackage main\n\nimport (\n\t\"os\"\n\n\t\"github.com/pulumi/pulumi-hcloud/sdk/go/hcloud\"\n\t\"github.com/pulumi/pulumi/sdk/v3/go/pulumi\"\n)\n\nfunc readFileOrPanic(path string) pulumi.StringPtrInput {\n\tdata, err := os.ReadFile(path)\n\tif err != nil {\n\t\tpanic(err.Error())\n\t}\n\treturn pulumi.String(string(data))\n}\n\nfunc main() {\n\tpulumi.Run(func(ctx *pulumi.Context) error {\n\t\t// Create a new SSH key\n\t\t_, err := hcloud.NewSshKey(ctx, \"default\", \u0026hcloud.SshKeyArgs{\n\t\t\tPublicKey: readFileOrPanic(\"~/.ssh/id_rsa.pub\"),\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\treturn nil\n\t})\n}\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.hcloud.SshKey;\nimport com.pulumi.hcloud.SshKeyArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        // Create a new SSH key\n        var default_ = new SshKey(\"default\", SshKeyArgs.builder()        \n            .publicKey(Files.readString(Paths.get(\"~/.ssh/id_rsa.pub\")))\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  # Create a new SSH key\n  default:\n    type: hcloud:SshKey\n    properties:\n      publicKey:\n        fn::readFile: ~/.ssh/id_rsa.pub\n```\n\u003c!--End PulumiCodeChooser --\u003e\n\n## Import\n\nSSH keys can be imported using the SSH key `id`:\n\n```sh\n$ pulumi import hcloud:index/sshKey:SshKey mykey id\n```\n\n",
+            "description": "Provides a Hetzner Cloud SSH key resource to manage SSH keys for server access.\n\n## Import\n\nSSH keys can be imported using the SSH key `id`:\n\n```sh\n$ pulumi import hcloud:index/sshKey:SshKey mykey id\n```\n\n",
```

Comment:
Here we lost all examples except the import?

Reply:
This is back now.